### PR TITLE
Add diffusers version and pipeline class to the Hub UA

### PR DESCRIPTION
**Pull request by anton-l**
This will let us know which versions and pipelines are used the most, which in turn informs the deprecation cycles and maintenance priority. Kudos to @LysandreJik for the tip!

### CODE REVIEW

File: src/diffusers/pipeline_utils.py [hunk 1]
```diff
@@ -373,6 +374,11 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
             if custom_pipeline is not None:
                 allow_patterns += [CUSTOM_PIPELINE_FILE_NAME]
 
+            requested_pipeline_class = config_dict.get("_class_name", cls.__name__)
```
Comment (anton-l) [inline src/diffusers/pipeline_utils.py:377]: This should cover all cases, right?

File: src/diffusers/modeling_utils.py [hunk 2]
```diff
@@ -297,7 +298,12 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         subfolder = kwargs.pop("subfolder", None)
         device_map = kwargs.pop("device_map", None)
 
-        user_agent = {"file_type": "model", "framework": "pytorch", "from_auto_class": from_auto_class}
+        user_agent = {
+            "diffusers": __version__,
+            "file_type": "model",
+            "framework": "pytorch",
+            "from_auto_class": from_auto_class,
```
Comment (patrickvonplaten) [inline src/diffusers/modeling_utils.py:305]:
```suggestion
```
Think we don't really have the `from_auto_class` features yet (we should pass this via the `DiffusionPipeline` class eventually, but maybe in a next PR? :-))

File: src/diffusers/pipeline_utils.py (hunk 1 above)
Comment (patrickvonplaten) [reply src/diffusers/pipeline_utils.py:377]: Yes that'll be a very powerful metric - nice!

File: src/diffusers/modeling_utils.py (hunk 2 above)
Comment (anton-l) [reply src/diffusers/modeling_utils.py:305]: Ah, this got inherited when we were copying the model loaders from transformers. Should be removed indeed :)

**Comment by anton-l** [comment]
I think the autoclass param sneaked in when we were copying the model loaders from transformers. Should be removed indeed :)

**Comment by sgugger** [comment]
I'm really not sure what you want me to have a look at here @patrickvonplaten . Is there a link to Transformers or Accelerate?

**Review by LysandreJik — APPROVED** [review]
Nice! You could try using that branch to load a few models and see if you can find them in Kibana

**Comment by anton-l** [comment]
Yes, got some hits!

**Pull request merged**